openapi.yaml

```diff
@@ -211,6 +211,9 @@ paths:
         405:
           description: Validation exception
           content: {}
+        409:
+          description: Conflict with current state
+          content: {}
```

Comment:
I feel like using an empty content tag here is not necessary, From what I can tell, it adds a little `media type` tag that does not provide any functionality. I don't think you should remove it everywhere we do this, but I am somewhat against adding that if not necessary.

Reply:
I agree. I removed the empty content tag from the 409 status codes that I had previously included.